test/stage.spec.js

```diff
@@ -332,6 +332,19 @@ describe('stage', () => {
       expect(appRenderer.resolution).toEqual(2)
     })
 
+    test('does not update resolution of interaction plugin if interaction plugin is removed', () => {
+      delete PIXI.Renderer.__plugins.interaction
```

Comment:
After the test we should reset the interaction manager again

Reply:
Something like:

```js
        const im = PIXI.Renderer.__plugins.interaction
        delete PIXI.Renderer.__plugins.interaction
      
       ...

        PIXI.Renderer.__plugins.interaction = im
```

Reply:
That's reasonable! I've just made the fix.